tests/code_wars/test_scramble.py

```diff
@@ -1,15 +1,35 @@
 from code_challenges.code_wars.scramble import count_characters, scramble
 
 
-def test_scramble_true():
-    actual = scramble("rkqodlw", "world")
-    expected = True
+def test_count_characters():
+    actual = count_characters("rkqodlw", "world")
+    expected = {"r": 1, "k": 1, "q": 1, "o": 1, "d": 1, "l": 1, "w": 1}, {
+        "w": 1,
+        "o": 1,
+        "r": 1,
+        "l": 1,
+        "d": 1,
+    }
```

Comment:
Info only, no need to change:  If you wanted these to format better you could write this as:
```python
    expected = (
        {"r": 1, "k": 1, "q": 1, "o": 1, "d": 1, "l": 1, "w": 1},
        {"w": 1, "o": 1, "r": 1, "l": 1, "d": 1},
    )
```

Reply:
great info cuz thats how i wanted to do it and wasn't sure how! thank you!!!